docs/User-Guides/LDR-R1 VM Deployment.md

```diff
@@ -69,26 +69,27 @@ Checklist:
     ## Sample config.ini for single node VM
     ```
     [storage_enclosure]
-    type=other
+    type=virtual
 
     [srvnode-1]
     hostname=host1.localdomain
-    network.data_nw.pvt_ip_addr=None
-    network.data_nw.iface=eth1, eth2
+    network.data_nw.pvt_ip_addr=192.254.254.254
```

Comment:
I think this is a typo.
it should be None

Reply:
This is required for ova.